package.json

```diff
@@ -1,25 +0,0 @@
-{
-  "name": "projeto12-batepapo-uol-api",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "type": "module",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "nodemon back-bate-papo-uol/index.js"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "chalk": "^5.0.1",
-    "cors": "^2.8.5",
-    "dayjs": "^1.11.1",
-    "dotenv": "^16.0.0",
-    "express": "^4.18.0",
-    "joi": "^17.6.0",
-    "mongodb": "^4.5.0",
-    "nodemon": "^2.0.15",
```

Comment:
Um ponto de atenção é sobre as **dependencies** que são os pacotes que precisam estar no projeto para que ele funcione e **devDependencies** que são os pacotes que facilitam o teste local ou rodar localmente como é o caso do **nodemon**.

Instalamos no devDependencies usando a seguinte sintaxe **npm install thePackageName --save-dev**

fonte: https://levitrares.com/host-https-qastack.com.br/programming/36999461/how-to-install-only-devdependencies-using-npm